src/physics/arcade/World.js

```diff
@@ -1920,7 +1920,7 @@ var World = new Class({
     {
         var bodyA = sprite.body;
 
-        if (group.length === 0 || !bodyA || !bodyA.enable)
+        if (group.length === 0 || !bodyA || !bodyA.enable || bodyA.checkCollision.none)
```

Comment:
Is it worth adding the same check for `bodyB` inside the loop below, or am I missing something?

Reply:
I suppose so.

Reply:
Pushed this change for `bodyB`.

Reply:
Looks good!